lib/msal-browser/test/cache/BrowserCacheManager.spec.ts

```diff
@@ -944,6 +944,92 @@ describe("BrowserCacheManager tests", () => {
                         expect(browserLocalStorage.getAccessTokenCredential(testAccessTokenWithAuthScheme.generateCredentialKey())).toBeInstanceOf(AccessTokenEntity);
                     }
                 )
+
+                it.only("clearTokensWithClaimsInCache clears all access tokens with claims in tokenKeys", () => {
```

Comment:
```suggestion
                it("clearTokensWithClaimsInCache clears all access tokens with claims in tokenKeys", () => {
```

Reply:
I'll check to see if there's a linter rule for this or something, I check in onlies a lot.

Reply:
Thanks @hectormmg. Yes, after many many test runs, forgot to take this out.